doc/ref/plot_options/size.ipynb

```diff
@@ -0,0 +1,262 @@
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2ae44fa2-2ef0-4204-b9bb-b6051dfefd7c",
+   "metadata": {},
+   "source": [
+    "# Size and Layout Options\n",
+    "\n",
+    "\n",
+    "```{eval-rst}\n",
+    ".. plotting-options-table:: Size And Layout Options\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2acaf31a-0348-4c0e-a486-ab165e59725b",
+   "metadata": {},
+   "source": [
+    "## `fontscale`\n",
+    "\n",
+    "The `fontscale` option scales all the fonts in the plot by the provided numeric factor. For example, setting `fontscale=1.5` enlarges the title, tick labels, and axis labels by 50%. This is useful when you want to emphasize text for presentations or detailed viewing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f04a7fc8-687b-4ad7-b01f-ac102e5a5a4e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "df.hvplot.scatter(\n",
+    "    x='bill_length_mm',\n",
+    "    y='bill_depth_mm',\n",
+    "    by='species',\n",
+    "    fontscale=1.5,\n",
+    "    title=\"Penguins Species\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c99dceda-b76f-450a-90ac-06735a1b001c",
+   "metadata": {},
+   "source": [
+    "## `frame_width / frame_height`\n",
+    "\n",
+    "The `frame_width` and `frame_height` options determine the width and height of the data area within the plot. They define the size of the plot’s core region (excluding axes, legends, and margins), allowing precise control over how the data is displayed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1e5090c0-4687-477e-9603-42065aaf2791",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    by='mag_class',\n",
+    "    tiles=True,\n",
+    "    title=\"Earthquakes in the pacific\",\n",
+    "    frame_width=600,\n",
+    "    frame_height=400\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16e0b4c9-1648-4087-82b9-0f1e3b064cd0",
+   "metadata": {},
+   "source": [
+    "## `max_width / max_height`\n",
+    "\n",
+    "The `max_width` and `max_height` options set the maximum allowed dimensions for the plot. These options come into play in responsive or dynamic layouts to ensure that the plot does not grow beyond the specified limits when the browser window is resized."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a6fbe571-c429-496f-809d-9f304f4773f5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.xarray  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.air_temperature(\"xarray\")\n",
```

Comment:
Pick just one timestamp to avoid the non-interactive widgets.

Reply:
OK